.travis.yml

```diff
@@ -3,6 +3,10 @@ node_js:
 - '5.6'
 - '4.3'
 - '0.12'
+notifications:
+  email:
+    on_success: never
```

Comment:
Good question, I guess most of the time you'll be watching travis after you have a failed build?